pages/vi/vi-docker-tutorial.md

```diff
@@ -254,4 +254,13 @@ Commands:
   version            Show the Docker-Compose version information
 ```
 
+## Useful Links
+
+[Why install Docker?](https://www.docker.com/why-docker) 
+[Docker Concepts](https://docs.docker.com/get-started/#docker-concepts)
+[Docker Overview](https://docs.docker.com/engine/docker-overview/)
+[Docker Compose](https://docs.docker.com/compose/overview/)
+[Wikipedia page on Docker](https://en.wikipedia.org/wiki/Docker_(software)
+[Other useful Docker tutorial](https://scotch.io/tutorials/getting-started-with-docker)
```

Comment:
We already have added links with docker cli command so use that
`https://docs.docker.com/engine/reference/commandline/cli/`

Reply:
will do